sw/device/lib/dif/dif_plic.c

```diff
@@ -44,85 +52,28 @@ typedef struct plic_target_reg_offset {
   ptrdiff_t threshold; /*<< Threshold register offset. */
 } plic_target_reg_offset_t;
 
-/**
- * PLIC peripheral IRQ range.
- *
- * PLIC IRQ source IDs are grouped together by a peripheral they belong to.
- * Meaning that all IRQ IDs of the same peripheral are guaranteed to be
- * consecutive. This data type is used to store IRQ ID ranges of a peripheral.
- */
-typedef struct plic_peripheral_range {
-  dif_plic_irq_id_t first_irq_id; /*<< The first IRQ ID of a peripheral. */
-  dif_plic_irq_id_t last_irq_id;  /*<< The last IRQ ID of a peripheral. */
-} plic_peripheral_range_t;
-
 // An array of target specific set of register offsets. Every supported PLIC
 // target must have an entry in this array.
 static const plic_target_reg_offset_t plic_target_reg_offsets[] = {
-        [kDifPlicTargetIbex0] =
+        [0] =
```

Comment:
It would be nice to explain the relationship between unnamed targets and the "outside dif", but I am not sure what is the best way? Maybe at the top of the file a short paragraph?

Reply:
Do you mean as in what constitutes a "target"?

I'll add some more details in the documentation for `dif_plic_target_t`. 

I've also expanded documentation for this specific table, to avoid talking about external targets (such as Ibex cores), and only talk about how target IDs map to register offsets.

Reply:
Thanks, looks good.